include/grpc/grpc_security_constants.h

```diff
@@ -29,6 +29,15 @@ extern "C" {
 #define GRPC_X509_CN_PROPERTY_NAME "x509_common_name"
 #define GRPC_X509_SAN_PROPERTY_NAME "x509_subject_alternative_name"
 #define GRPC_X509_PEM_CERT_PROPERTY_NAME "x509_pem_cert"
+// Please note that internally, we just faithfully pass whatever value we got by
+// calling SSL_get_peer_certificate() in OpenSSL/BoringSSL. This will mean in
+// OpenSSL, the following conditions might apply:
+// 1. on the server side, the full chain would be the whole chain without the
+// leaf certificate
```

Comment:
I would rephrase

One the client side, this property returns the full certificate chain. On the server side, this property will return the certificate chain without the leaf certificate. Application can use GRPC_X509_PEM_CERT_PROPERTY_NAME to get the peer leaf certificate.